hugegraph-dist/docker/example/docker-compose-cassandra.yml

```diff
@@ -0,0 +1,44 @@
+version: "3"
+
+services:
+  graph:
+    image: hugegraph/hugegraph
+    container_name: ca-graph
+    ports:
+      - 18080:8080
+    environment:
+      hugegraph.backend: cassandra
+      hugegraph.serializer: cassandra
+      hugegraph.cassandra.host: ca-cassandra
+      hugegraph.cassandra.port: 9042
+    networks:
+      - ca-network
+    depends_on:
+      - cassandra
+    healthcheck:
+      test: ["CMD", "bin/gremlin-console.sh", "--" ,"-e", "scripts/remote-connect.groovy"]
+      interval: 10s
+      timeout: 30s
+      retries: 3
+
+  cassandra:
+    image: cassandra:3.11
```

Comment:
try C* 4 for it

Reply:
tested, no problem
![image](https://github.com/apache/incubator-hugegraph/assets/49650772/2e732c44-54cd-4c98-bef0-4c56be1b694c)